src/client/cockpit-beiboot

```diff
@@ -220,8 +223,14 @@ class SshPeer(Peer):
         logger.debug("Launching command: cmd=%s env=%s", cmd, env)
         transport = await self.spawn(cmd, env, stderr=agent, start_new_session=True)
 
+        if not self.always:
+            usr_bin_cockpit_bridge_steps = [('try_exec', (['/usr/bin/cockpit-bridge'],))]
```

Comment:
Could we please drop the hardcoded path? `sudo make install` installs into /usr/local, and we even used to do that in our /pybridge scenario. This ought to work with `$PATH`, no?

Or will this conflict with the flatpak and /apps/ etc.? Is that in $PATH?